src/libUtils/Logger.cpp

```diff
@@ -377,3 +396,18 @@ Logger::ScopeMarker::~ScopeMarker() {
         << " END";
   }
 }
+
+std::string_view LogTracesImpl() {
+  static thread_local char BUFFER[TRACE_INFO_BUFSIZE];
```

Comment:
nitpick, std::array<> ?

Reply:
I need to take this code immediately, we will flag this and fix it in a new PR.